README.md

```diff
@@ -226,7 +226,7 @@ A few more notes:
   DOCKER_BUILD_OPTS=--build-arg UBUNTU_VERSION=18.04 --build-arg UBUNTU_NAME=bionic
   ```
 * If you run behind a proxy, you will have to configure the proxy,
-  e.g., for docker (`~/.docker/config.json`). See
+  e.g., for docker (`~/.docker/config.json`) and load the configuration inside the dev container by setting `DOCKER_DEV_RUN_OPTS += -v "~/.docker":"/root/.docker"` in `$FPC_PATH/config.override.mk`. See
```

Comment:
@mbrandenburger have you seen the comments above?

Reply:
ah we should propose to use `"$HOME/.docker"` instead of '~'.

Reply:
anyway, the second error message is expected if you don't switch the docker context. 

Maybe we need to update the wording of the readme saying that this might introduce additional complications?!

Reply:
let me try with `$HOME/.docker`

Reply:
@mbrandenburger I had to use DOCKER_DEV_RUN_OPTS += -v $$HOME/.docker:/root/.docker for it to work

Reply:
so I also tried without having this configuration set and everything worked, including running `make docker` once inside the container

Reply:
So, finally tested without this configuration, and after having changed docker desktop configuration to use the legacy implementation for the file sharing I was able to follow thu with the readme.
If I use this configuration, once I reach the Build my first chaincode and try to execute `make docker` I get the error that is solved changing the docker context used.